Client/src/components/contanier/student/Profile/index.js

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react'
+import TextCard from '../../../common/TextCard'
+import './index.css';
 
 export default class Profile extends Component {
+  state = {
+    level: 6,
+  }
+
   render() {
+    const label = ['Name :', 'Phone Number :', 'Amideast ID :']
```

Comment:
I think that if you put this in the state, you can simplify your work when you fetch the data !!

Reply:
Done